frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPaletteTriggerButton/CommandPaletteTriggerButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Search } from '@liam-hq/ui'
+import type { FC } from 'react'
+import { useCommandPalette } from '../CommandPaletteProvider'
+import styles from './CommandPaletteTriggerButton.module.css'
+
+export const CommandPaletteTriggerButton: FC = () => {
+  const { setOpen } = useCommandPalette()
+
+  return (
+    <button
+      className={styles.container}
+      type="button"
+      onClick={() => setOpen(true)}
+      aria-label="Open command palette to search features"
```

Comment:
Withou this aria-label, the accessible name of this button will be Search ⌘ K"", but I don't think it makes sense.